docs/en/observability/apm/tab-widgets/install-agents.asciidoc

```diff
@@ -223,91 +217,74 @@ Read more in the {apm-ios-ref}/intro.html[APM iOS Agent Reference].
 
-*Start your application with the `javaagent` flag*
+- download manually the link:https://oss.sonatype.org/service/local/artifact/maven/redirect?r=releases&g=co.elastic.apm&a=elastic-apm-agent&v=LATEST[latest agent]
+or link:https://mvnrepository.com/artifact/co.elastic.apm/elastic-apm-agent[previous releases] from Maven central.
+- download with `curl`:
++
+[source,bash]
+----
+curl -o 'elastic-apm-agent.jar' -L 'https://oss.sonatype.org/service/local/artifact/maven/redirect?r=releases&g=co.elastic.apm&a=elastic-apm-agent&v=LATEST'
+----
 
-Add the `-javaagent` flag and configure the agent with system properties.
+*Add `-javaagent` flag*
 
-* Set required service name
-* Set custom APM Server URL (default: http://localhost:8200)
-* Set the base package of your application
+When starting your application, add the JVM flag `-javaagent:/path/to/elastic-apm-agent-<version>.jar`
 
-[source,java]
+*Generic setup*
+
+Start your application (for example a Spring Boot application or other embedded servers) and add the `-javaagent` JVM flag.
+Use the `-D` prefix to configure the agent using system properties.
+
+[source,bash]
 ----
-java -javaagent:/path/to/elastic-apm-agent-<version>.jar \
-     -Delastic.apm.service_name=my-application \
-     -Delastic.apm.server_urls=http://localhost:8200 \
-     -Delastic.apm.secret_token= \
-     -Delastic.apm.application_packages=org.example \
-     -jar my-application.jar
+java -javaagent:/path/to/elastic-apm-agent-<version>.jar -Delastic.apm.service_name=my-cool-service -Delastic.apm.application_packages=org.example,org.another.example -Delastic.apm.server_url=http://127.0.0.1:8200 -jar my-application.jar
 ----
 
 *Learn more in the agent reference*
 
+* {apm-java-ref-v}/setup.html[Complete setup instructions]
 * {apm-java-ref-v}/supported-technologies-details.html[Supported technologies]
-* {apm-java-ref-v}/configuration.html[Advanced configuration]
+
 // end::java[]
 
 // ***************************************************
 // ***************************************************
 
 // tag::net[]
-*Download the {apm-agent}*
-
-Add the agent packages from https://www.nuget.org/packages?q=Elastic.apm[NuGet] to your .NET application.
-There are multiple NuGet packages available for different use cases.
+*Set up the {apm-agent}*
 
-For an ASP.NET Core application with Entity Framework Core, download the
-https://www.nuget.org/packages/Elastic.Apm.NetCoreAll[Elastic.Apm.NetCoreAll] package.
-This package will automatically add every agent component to your application.
+The .NET agent can be added to an application in three different ways
 
-To minimize the number of dependencies, you can use the
-https://www.nuget.org/packages/Elastic.Apm.AspNetCore[Elastic.Apm.AspNetCore] package for just ASP.NET Core monitoring, or the
-https://www.nuget.org/packages/Elastic.Apm.EntityFrameworkCore[Elastic.Apm.EfCore] package for just Entity Framework Core monitoring.
+Profiler runtime instrumentation::
```

Comment:
Maybe use an unordered list here like suggested in the previous comment? Or use description lists for both?

Reply:
Fixed in b4f3240